### PR TITLE
Added manual testing/QA doc

### PR DESCRIPTION
**Pull request by indirectlylit**
Re-opening @whitzhu's PR - https://github.com/learningequality/kolibri/pull/454

cc @radinamatic

### CODE REVIEW

**Comment by radinamatic** [comment]
Yey...! :smiley: 

Do you want to merge this already so I can add the paragraph about a11y testing?

**Comment by indirectlylit** [comment]
I'm just about to add another commit putting it in the TOC, and adding some notes about cross browser. Then I'll kick it over to you.

thanks!

**Comment by indirectlylit** [comment]
All right, ready for your review & merge

File: docs/dev/getting_started.rst [hunk 1]
```diff
@@ -211,8 +232,8 @@ To run specific tests only, you can add ``--``, followed by a label (consisting
   kolibri manage test -- kolibri.auth.test.test_permissions.MembershipPermissionsTestCase.test_admin_can_delete_membership
 
 
-Documentation Editing
-~~~~~~~~~~~~~~~~~~~~~
+Updating Documentation
+~~~~~~~~~~~~~~~~~~~~~~
 
 To make changes to documentation, make an edit and then run:
```
Comment (radinamatic) [inline docs/dev/getting_started.rst:245]: Maybe add running the `pip install -r requirements/docs.txt` too?

File: docs/dev/manual_testing.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+**Amazon Workspaces**
+
+In some situations, simply having a browser is not enough. For example, a developer may need to test Windows-specific backend or installer code from another OS. In many situations, a virtual machine is appropriate - however these can be slow to download and run.
+
+Amazon's `AWS Workspaces <https://aws.amazon.com/workspaces/>`_ provides a faster alternative. They run Windows VMs in their cloud, and developers can RDP in.
+
+**Local Virtual Machines**
+
+Workspaces is very useful, but it has limitations: only a small range of OSes are available, and connectivity and provisioning are required. An alternative is to run the guest operating system inside a virtual machine using e.g. `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_. This also gives more developer flexibility, including e.g. shared directories between the guest and host systems.
+
```
Comment (radinamatic) [inline docs/dev/manual_testing.rst]: I just remembered I wrote this[ tutorial for testing KA Lite in VirtualBox](https://docs.google.com/document/d/10LgeCJmqsweui0yTTCDf4DjY5aoNNpXG8hF_DGKUHAI/edit), back when I started the QA. It is in our FLE Shared folder, but it is viewable to general public...

File: docs/dev/manual_testing.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Responsiveness to Varying Screen Sizes
+--------------------------------------
+
+We want to ensure that the app looks and behaves reasonably across a wide range of typical screen sizes, from small tablets to large, HD monitors. It is highly recommended to constantly be testing functionality at a range of sizes.
+
+Chrome's Developer Tools has some excellent functionality to simulate arbitrary screen resolutions.
+
+
```
Comment (radinamatic) [inline docs/dev/manual_testing.rst:46]: Add:

In Firefox you can use `Developer Tools - Responsive Design View`.

**Comment by indirectlylit** [comment]
updates pushed

**Comment by radinamatic** [comment]
Ok, I don't really have write access here, so somebody else will have to merge... :sunglasses:

**Comment by indirectlylit** [comment]
gracias.

That's odd about the permissions - I thought everyone on the team could merge. I'll have to check the settings and discuss with aron & jamie...

**Pull request merged**